doc/content/reference/components/network-operations-center.md

```diff
@@ -11,6 +11,109 @@ Network Operations Center provides aggregated insight in a network operated with
 
 Network Operations Center stores events from {{% tts %}} in a time series database, queries aggregated data and visualizes data in a dashboard driven by Grafana. Network Operations Center is complementary to the [Console]({{< relref "console" >}}): the Network Operations Center shows historic data (with a delay of at most 5 minutes), while the Console shows live information and allows for managing the network.
 
-{{< figure src="../network-operations-center_overview.png" alt="Network Operations Center" >}}
+## Accessing Network Operations Center
+
+The Network Operations Center can only be accessed by administrators. There are two main ways to access Network Operations Center within {{% tts %}} console:
+
+- On the vertical navigation click Network Operations Center.
+- On the horizontal navigation click on your avatar/profile name and then from the drop-down menu click Network Operations Center.
+
+The Network Operations Center Overview dashboard will open in a new web browser tab.
+
+{{< note "Network Operations Center is supported in the current version of these [web browsers](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)." />}}
```

Comment:
```suggestion
{{< note "Since it utilizes Grafana, the Network Operations Center is supported in the current version of the following [web browsers](https://grafana.com/docs/grafana/latest/setup-grafana/installation#supported-web-browsers)." />}}
```